packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.hook.tsx

```diff
@@ -51,10 +51,12 @@ export function useDeleteVisit(patientUuid: string, visit: Visit, onVisitDelete
             : t('visitCancelled', 'Visit cancelled'),
           kind: 'success',
           subtitle: !isCurrentVisitDeleted
-            ? t('visitCancelSuccessMessage', 'Active {{visit}} cancelled successfully', {
+            ? t('visitDeletedSuccessfully', '{{visit}} was deleted successfully', {
```

Comment:
```suggestion
            ? t('visitDeletedSuccessfully', '{{visit}} deleted successfully', {
```

From the [design](https://zeroheight.com/23a080e38/p/683580-notifications), it looks like the verb (`was` in this case) is superfluous.

Reply:
Done!